docs/modules/ROOT/pages/user-guide/testing.adoc

```diff
@@ -226,3 +226,20 @@ class MyTest {
 ----
 
 More examples of WireMock usage can be found in the Camel Quarkus integration test source tree such as https://github.com/apache/camel-quarkus/tree/main/integration-tests/geocoder[Geocoder].
+
+== Testing with `CamelTestSupport` in JVM mode
```

Comment:
The title is a bit misleading IMO, because CamelTestSupport does not work on Quarkus. 

How about "`CamelTestSupport` style of testing" or "Testing with `CamelQuarkusTestSupport` in JVM mode"? 

Also in the introductory sentence, I think it would be good to give some context by including link to the page where CamelTestSupport is documented. E.g. 

> If you used plain Camel before, you may know `https://camel.apache.org/components/latest/others/test-junit5.html[CamelTestSupport]` already. Unfortunately the Camel variant won't work on Quarkus and so we prepared a replacement called `CamelQuarkusTestSupport`.

Reply:
Makes sense, thanks!